patterns/fluentbit-containerinsights/README.md

```diff
@@ -0,0 +1,337 @@
+# AWS for FluentBit with ContainerInsights and Kubelet monitoring.
```

Comment:
I'll offer an alternative thought - what about showing this as a snippet? 

1. Most of the detail is in the README (and its great info), very little has to do with the Terraform
2. I suspect there are a number of different logging configurations/scenarios that users may inquire about
3. There isn't anything specific to the cluster itself

What do you think?